hledger-lib/Hledger/Query.hs

```diff
@@ -177,9 +178,13 @@ data QueryOpt = QueryOptInAcctOnly AccountName  -- ^ show an account register fo
 -- >>> parseQuery nulldate "\"expenses:dining out\""
 -- Right (Acct (RegexpCI "expenses:dining out"),[])
```

Comment:
doctest needs update I think

Reply:
or removal

Reply:
I agree that these doctests don't make a lot of sense for `parseQueryList`, but I think as long as we're leaving `parseQuery` in we should be sure it's dealing with quoting issues correctly. On the other hand, if you're in favour of just removing `parseQuery` altogether (and renaming `parseQueryList`), then we could get rid of these.